sentry.gradle

```diff
@@ -44,7 +44,7 @@ gradle.projectsEvaluated {
         if (reactRoot == null) {
             reactRoot = props.get("root").get() // RN 0.71 and above
         }
-        def modulesOutput = "$reactRoot/android/app/src/main/assets/modules.json"
+        def modulesOutput = "$projectDir/src/main/assets/modules.json"
```

Comment:
This will work for the modules, but I think the issue is with the `$reactRoot` not being set to the root of your react native application project.

Because we use `$reactRoot` at other places like https://github.com/getsentry/sentry-react-native/blob/b093f67712a98bc80b03d5f85b58b84fb16f1adb/sentry.gradle#L103 and I assume those would not work for you too?

Reply:
I wonder if this is just a misconfiguration on our monorepo then?

The [`app/build.gradle` says](https://github.com/facebook/react-native/blob/main/packages/react-native/template/android/app/build.gradle#L11):

```gradle
//   The root of your project, i.e. where "package.json" lives. Default is '..'
```

Now, I read that as location 1 below but it sounds like you're saying it should be 2?

```
.
└── monorepo/ (1)
    ├── node_modules/
    │   ├── @sentry/
    │   │   └── react-native
    │   ├── react-native
    │   └── something-shared
    └── packages/
        └── an-app/ (2)
            └── node_modules/
                └── something-unique
```

Maybe I need to clarify with the RN maintainers... 🤔

Reply:
@liamjones Have you found out some new information? What tool are you using to manage/create the monorepo?

Reply:
With my altered setup I now have `react.root` and `react.entryFile` set to the default in `app/build.gradle`, as you were expecting, and just an override for `react.reactNativeDir`.